### PR TITLE
WRP-550: Write Docs about using debug aria feature

### PR DESCRIPTION
**Pull request by sjro**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
 Need to guide app developers how to use debug aria feature

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add the docs for debug aria feature

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-550

### Comments
Enact-DCO-1.0-Signed-off-by: SJ RO (sj.ro@lge.com)

### CODE REVIEW

**Review by juwonjeong — APPROVED** [review]
LGTM

File: docs/testing-components/index.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ title: Testing Your Components
 
 *   [Unit Testing](./unit-testing/index.md) - How to start Unit Testing
 *   [Test Driven Development(TDD)](./test-driven-development/index.md) - This document describes the test-first methodology
-we aspire to on the Enact team.
\ No newline at end of file
+*   [Debug Accessibility in Enact](./debug-aria/index.md) - How to debug accessibility in Enact app and sampler
```
Comment (seunghoh) [inline docs/testing-components/index.md:7]: This is not the testing feature. Let's move the new page to the bottom of the `Accessibility Support in Enact` page.

File: docs/testing-components/debug-aria/Debugaria-Sampler.png
Comment (seunghoh) [inline docs/testing-components/debug-aria/Debugaria-Sampler.png:1]: No hand-written red box. Please refer to the https://medium.com/enact-js/debugging-an-enact-app-with-vs-code-17a11049fd8d what we red-box added.

File: docs/testing-components/debug-aria/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Debug Accessibility in Enact
+---
+
+We explained that accessibility can be implemented in Enact in [Accessibility Support in Enact](../../accessibility) section.
```
Comment (seunghoh) [inline docs/testing-components/debug-aria/index.md:5]: We don't need this line when we move this section under the same page.

File: docs/testing-components/debug-aria/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Enact also provides ways to debug these accessibility implementations.
+
+## In sampler
+
+We provide a sampler in [UI COMPONENTS](https://enactjs.com/sampler/sandstone) that implements several UI components as [Storybook](https://storybook.js.org/).
+If click the `debug aria` tab located at the top of the page of the sampler and select `true`, the `aria-label` of each component is displayed as shown in the image below.
```
Comment (seunghoh) [inline docs/testing-components/debug-aria/index.md:11]: `aria-label` -> ARIA properties

File: docs/testing-components/debug-aria/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+![debug-aria in sampler](Debugaria-Sampler.png)
+
+## In Enact app
+
+This debug feature is designed to work if a class `debug aria` is given to any enact app.
+So just add `debug aria` to className prop in your enact app to get the current aria-label.
```
Comment (seunghoh) [inline docs/testing-components/debug-aria/index.md:18]: to get the current aria-label.
->  to see the applied ARIA properties.

File: docs/testing-components/debug-aria/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+const AppBase = (props) => {
+	return (
+		<div {...props}>
+			...
+		</div>
+	);
+};
+const debugAppBase = () => {
+	return (
+		<AppBase className='debug aria' />
```
Comment (seunghoh) [inline docs/testing-components/debug-aria/index.md:34]: Q. Don't we have any sample that condition check on debug mode only enabled?

**Review by seunghoh — CHANGES_REQUESTED** [review]

File: docs/accessibility/index.md [hunk 6]
```diff
@@ -87,3 +87,40 @@ const App = () => {
...
+const AppBase = (props) => {
+	return (
+		<div {...props}>
+			...
+		</div>
+	);
+};
+const debugAppBase = () => {
+	return (
+		<AppBase className='debug aria' />
```
Comment (seunghoh) [inline docs/accessibility/index.md:121]: Again, this is the question. 
Don't we have any sample that condition check on debug mode only enabled?

File: docs/accessibility/index.md [hunk 7]
```diff
@@ -87,3 +87,40 @@ const App = () => {
 	);
 };
 ```
+
+## Debug Accessibility
+
+Enact also provides ways to debug these accessibility implementations.
+
+### In sampler
```
Comment (seunghoh) [inline docs/accessibility/index.md:95]:
```suggestion
### In Sampler
```

File: docs/accessibility/index.md (hunk 6 above)
Comment (sjro) [reply docs/accessibility/index.md:121]: I remained the answer in ticket comments.

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**